Frameworks/UIKit/StarboardXaml/ApplicationMain.mm

```diff
@@ -179,6 +179,6 @@ int ApplicationMainStart(const char* principalName,
     return 0;
 }
 
-void SetTemporaryFolder(const char* folder) {
-    NSSetTemporaryDirectory([NSString stringWithCString:folder]);
+void SetTemporaryFolder(const wchar_t* folder) {
+    NSSetTemporaryDirectory([NSString stringWithCharacters:(const unichar*)folder length:wcslen(folder)]);
```

Comment:
@DHowett-MSFT yes, and in fact, we do get the value from a HSTRING.  But this being a temp folder, it is going to be copied when added to any file path.